src/workers/downloadPDF.ts

```diff
@@ -12,6 +12,10 @@ class JobData extends Job {
   }
 }
 
+function hashPdf(pdfBuffer: Buffer): string {
+  return crypto.createHash('sha256').update(pdfBuffer).digest('hex')
+}
+
```

Comment:
Do we want to use the same `hashPpf()` implementation for both `downloadPDF` and `pdf2Markdown` to avoid potential mismatch errors when we check the content in the future?